### DISCUSSION TITLE
How to add small images, icons in the hugo docsy

### DISCUSSION THREAD

**Discussion by arunnvkumar** [google/docsy#1033] [General]
**How to add small images, icons in the hugo docsy?**

Are there any command or short codes for adding inline images or icon size images?

**Comment by fekete-robert** [comment]
Hi, since Docsy already uses fontawesome, you can use those icons inline using simple HTML in your markdown content, like:

```
My sentence includes a fontawesome icon <i class="fas fa-arrow-alt-circle-right ml-2"></i> and continues.
```

If you want to add local images, then markdown images can be used inline as well, and will work if the image is small, like:
```
My sentence includes a local icon: ![Alt text for my icon](/img/my-icon.png)
```
